tests/integration/reaper_integration_test.go

```diff
@@ -0,0 +1,46 @@
+package integration
+
+import (
+	. "github.com/k8ssandra/k8ssandra/tests/integration/steps"
+
+	. "github.com/onsi/ginkgo"
+)
+
+var _ = Describe("Reaper", func() {
+	Context("Deploy K8ssandra with Reaper", func() {
+		It("Create a kind cluster and a namespace", func() {
+			AKindClusterIsRunningAndReachableStep("three workers")
+			IInstallTraefikStep()
+			ICreateTheNamespaceStep()
+			ICanSeeTheNamespaceInTheListOfNamespacesStep()
+		})
+		It("Install K8ssandra with Reaper", func() {
+			IDeployAClusterWithOptionsInTheNamespaceUsingTheValuesStep("default", "three_nodes_cluster_with_reaper.yaml")
+		})
+		It("Check the presence of expected resources", func() {
+			ICanCheckThatResourceOfTypeWithLabelIsPresentInNamespaceStep("service", "app.kubernetes.io/managed-by=reaper-operator")
+			ICanCheckThatResourceOfTypeWithNameIsPresentInNamespaceStep("service", "k8ssandra-dc1-all-pods-service")
+			ICanCheckThatResourceOfTypeWithNameIsPresentInNamespaceStep("service", "k8ssandra-dc1-service")
```

Comment:
Why couldn't I check? I know the function takes type, name as parameters. Simply: ``checkResourceExists("service", "kassandra-dc1-service")``

Reply:
Again, this is inherited from GoDog which generated function names based on the Gherkin steps.